modules/gdscript/gdscript_analyzer.cpp

```diff
@@ -1745,6 +1745,7 @@ void GDScriptAnalyzer::resolve_parameter(GDScriptParser::ParameterNode *p_parame
 		push_error(vformat(R"(Could not infer the type of the variable "%s" because the initial value is "null".)", p_parameter->identifier->name), p_parameter->default_value);
 	}
 
+	result.is_constant = false;
```

Comment:
There's code above setting this to `false` already, which is now redundant.

If this can never be constant, I'd move this up to where `result` is initialized.

Reply:
You're right about the redundancy, I removed the code above that is setting `result.is_constant = false`.

But if you take a look at lines [1722](https://github.com/godotengine/godot/pull/68985/files#diff-e01550d654cd3b28e2c1abaa393c95a08008ea22c5860cbd034b515b84635237R1722) and [1731](https://github.com/godotengine/godot/pull/68985/files#diff-e01550d654cd3b28e2c1abaa393c95a08008ea22c5860cbd034b515b84635237R1731), you'll see that `result` is assigned there, which makes moving `is_constant = false` up impossible.